packages/text-bitmap/src/BitmapFont.ts

```diff
@@ -226,4 +236,298 @@ export class BitmapFont
     {
         return available;
     }
+
+    /**
+     * Generates a bitmap-font for the given style and character set. This does not support
+     * kernings yet.
```

Comment:
when using this for meters, one of the biggest requirements is that each character takes up the same width. So when you're counting up numbers, they all stay in the same spot. If kerning isn't support yet, could there be a way to hack in some consistent kernings for, say, each numeric character, docced for now?

Reply:
@themoonrat Sorry on this one. I tried searching about kernings and how they work but I couldn't find any good documentation. If you could point me to something, that would be helpful 😃

Reply:
I'll take a look tomorrow :)